docs/platform/visual/index.md

```diff
@@ -57,92 +57,40 @@ they're customizable and extensible to fit your organization's desired look and
 
-If you provide any configuration file you must as well declare it as implementing `"pentaho.config.spec.IRuleSet"`.
+2. Your Bar/D3 visualization in the PDI menu:
+   
+   <img src="img/sample-bar-d3-pdi-menu.png" alt="Bar/D3 in PDI - menu" style="width: 365px;">
+   
+3. Your Bar/D3 visualization in PDI:
+   
+   <img src="img/sample-bar-d3-pdi.png" alt="Bar/D3 in PDI" style="width: 777px;">
 
-Finally, any third-party dependencies must be declared in the same file.
+<!-- TODO: Explain how to distribute it using marketplace? -->
 
-```json
-{ 
-  "name": "foo",
-  "version": "1.0.0",
-  "config": {
-    "pentaho/service": {
-      "foo_1.0.0/my-viz/model": "pentaho/visual/base",
-      "foo_1.0.0/my-viz/config": "pentaho.config.spec.IRuleSet"
-    }
-  },
-  "dependencies": {
-    "bar": "~2.0"
-  }
-}
-```
+# Configure the visualization
 
-See [Pentaho Web Package description](../web-package) for a more detailed view.
+Besides the default configuration you have included with the visualization,
+the visualization can be further configured by third-parties. 
 
-# Bundling and provisioning
-
-The Pentaho platform is built on top of an OSGi container, so developers must provide their code as an OSGi bundle. 
-Additionally, the required client side dependencies must also be provided to the platform as bundles.
-
-The recommended way is to put the visualization bundle, its dependencies, 
-and corresponding feature definition together into a single KAR file.
-
-See [Bundling a web package and provisioning](../bundling) for instructions.
-
-# Deploying the visualization
-
-The platform supports hot deployment: simply drop a bundle file in the deploy directory and 
-Apache Karaf will detect the file and try to deploy it.
-
-For PDI the Karaf folder is located in `system/karaf`. 
-On the Pentaho Server it's found within `pentaho-solutions/system/karaf`.
-
-You can drop any KAR file, bundle or Feature file into the `karaf/deploy` folder. 
-It will be automatically installed and activated, even after restarts of the product. 
-Replacing a bundle or feature already in the deploy folder will reinstall it within OSGI. Deleting it will uninstall.
-
-If every thing went well you should now see your visualization in Analyzer and/or PDI.
-
-<img src="img/sample-bar-d3-analyzer.png" alt="Bar/D3 in Analyzer" style="width: 100%; max-width: 575px;"><br>
-*Your Bar/D3 walkthrough visualization in Analyzer*
-
-<img src="img/sample-bar-d3-pdi-menu.png" alt="Bar/D3 in PDI - menu" style="width: 40%; max-width: 274px;  min-width: 274px;">
-<img src="img/sample-bar-d3-pdi.png" alt="Bar/D3 in PDI" style="width: 60%;max-width: 583px;  min-width: 420px;"><br>
-*Your Bar/D3 walkthrough visualization in PDI*
-
-> TODO: Explain how to distribute it using marketplace?
-
-# Configuring the visualization
-
-Pentaho's JavaScript API allows users to declare _prioritized configuration rules_ 
```

Comment:
This removed text, or a similar one, should be kept as an introduction to someone that reads quickly through this top page and didn't read the walkthrough part related with configuration.